lang/sucuri-scanner.pot

```diff
@@ -1,15 +1,15 @@
-# Copyright (C) 2019 Sucuri Inc.
+# Copyright (C) 2020 Sucuri Inc.
 # This file is distributed under the same license as the Sucuri Security - Auditing, Malware Scanner and Hardening plugin.
 msgid ""
 msgstr ""
 "Project-Id-Version: Sucuri Security - Auditing, Malware Scanner and Hardening 1.8.23\n"
-"Report-Msgid-Bugs-To: https://wordpress.org/support/plugin/sucuri-scanner\n"
+"Report-Msgid-Bugs-To: https://wordpress.org/support/plugin/sucuri-wordpress-plugin\n"
```

Comment:
"https://wordpress.org/support/plugin/sucuri-wordpress-plugin/" does not exist, "https://wordpress.org/support/plugin/sucuri-scanner/" is the correct URL.